commit 6f6bceb17d986b82a3a142f7063fd7a69c2a72db
Author: Michael Still <mikal@stillhq.com>
Date:   Fri Jun 5 19:16:56 2026 +1000

    Revert "Switch to Gazpacho/2026.1 sources".
    
    This reverts the OPENSTACK_RELEASE bump from kolla commit 352b7fce.
    The stable/2026.1 branch is now active, so the late-cycle pin to
    '2026.1' is stale and needs to be reverted back to 'master' --
    mirroring the same housekeeping pattern as the 2025.1->master
    revert (87678d014, 2025-05) and the 2025.2->master revert
    (c30e4e524, 2025-12).
    
    I specifically found this because Nova master commit 770580bb bumped
    oslo.privsep in requirements.txt from >=2.6.2 to >=3.11.0. With
    OPENSTACK_RELEASE = '2026.1', openstack_branch resolves to
    stable-2026.1 and openstack-base pulls
    requirements-stable-2026.1.tar.gz, which still pins
    oslo.privsep===3.10.1. The resulting ResolutionImpossible causes
    image builds to fail.
    
    Change-Id: I7d156af2fa9677cd4921f518ac1c6ecf7acde9ce
    Signed-off-by: Michael Still <mikal@stillhq.com>

diff --git a/kolla/common/config.py b/kolla/common/config.py
index 8a80e1f88..33938d5e2 100644
--- a/kolla/common/config.py
+++ b/kolla/common/config.py
@@ -38,7 +38,7 @@ DISTRO_PRETTY_NAME = {
     'ubuntu': 'Ubuntu 24.04.* LTS',
 }
 
-OPENSTACK_RELEASE = '2026.1'
+OPENSTACK_RELEASE = 'master'
 OPENSTACK_RELEASE_CODENAME = 'Gazpacho'
 
 # TODO(mandre) check for file integrity instead of downloading from an HTTPS
